website/docs/docs/dbt-cloud/access-control/self-service-permissions.md

```diff
@@ -18,3 +16,20 @@ dbt Cloud supports two different permission sets to manage permissions for self-
 | Manage billing | ❌ | ✅ |
 | Manage team permissions | ❌ | ✅ |
 | Invite Owners to the account | ❌ | ✅ |
+
+## Member and Owner Groups in dbt Cloud Enterprise
+
+By default, new users are added to the Member and Owner groups when they onboard to dbt Cloud. This is necessary for account onboarding but can create confusion when initially setting up SSO and RBAC for dbt Cloud Enterprise accounts as described in the [Enterprise Permissions](https://docs.getdbt.com/docs/dbt-cloud/access-control/enterprise-permissions/) guide. Users may be added with more permissions than intended without adjusting this setting.
+
+We recommend the following steps be taken after onboarding Administrative users and before adding less priveleged users to a dbt Cloud Enterprise account.
```

Comment:
Why not just have them delete the groups after SSO is setup?

Reply:
Some smaller teams might have an "everyone is an Owner" approach, which probably still isn't ideal. I'll add some additional text noting that they can delete the groups as well.

Reply:
@amychen1776 Would bet that some smaller groups will have an "everyone's an Owner" approach, so was thinking they may want to disable instead of remove outright. Definitely worth noting they can remove the groups as well so will add that to the text.

Reply:
Ah sorry Matt. I should have been more clear. They need to actually remove the groups outright after they have successfully created the Enterprise RBAC groups. They can't have the Owner and Member groups existing because it can cause weird bugs. The Owner permission set and the Account Admin permissions are completely the same so they need to delete the owner group and then create a new one with the Account Admin permission set. If they have a `everyone is an owner` mindset --> they still need to move everyone to the Account Admin permission set.

Reply:
Got it! Thanks for adding that detail.

Reply:
@amychen1776 would it be possible to include this information in the Multi-Tenant Deployment Plan? 

![image](https://user-images.githubusercontent.com/75497565/108740452-ebafb280-74f2-11eb-894d-12dff996c24f.png)

Looks like I'm read only on this one. If you want to grant me edit access I can add as well.